test/external/testcorenrn/CMakeLists.txt

```diff
@@ -3,15 +3,21 @@ include(NeuronTestHelper)
 set(direct_tests netstimdirect)
 set(spike_comparison_tests
     bbcore
-    conc
-    deriv
     gf
-    kin
     patstim
     vecplay
     watch
     vecevent)
 
+set(modfiles mod/Gfluct3.mod mod/hhwatch.mod mod/nacum.mod mod/vecevent.mod)
+if(NOT CORENRN_ENABLE_NMODL OR NOT CORENRN_ENABLE_GPU)
```

Comment:
Just a comment- somehow my brain gets confused and find this more straightforward to read : NOT ( ENABLE_NMODL & CORENEURON_GPU) 😅